.gitignore

```diff
@@ -0,0 +1,42 @@
+*.py[cod]
+
+# Testing cruft
+.tox
+.venv
+*.egg*
+egg
+.coverage
+cover
+nosetests.xml
+.testrepository
+.stestr
+edpm_ansible.egg-info/
+__pycache__
+build
+
+# Editors
+*~
+.*.swp
+.*sw?
+
+# Playbook retry files
+*.retry
+
+# roles
+edpm_ansible/roles.galaxy/*
+
+# Sphinx
+doc/build
+
+ansible-errors.json
+pytestdebug.log
+
+# doc
+docs/build/*
+
+# JetBrain
+.idea/
+.cache
+
+# Custom inventory variable files
+edpm_ansible/inventory/99-standalone-vars
```

Comment:
ok, maybe that one may be ignored, but it should be in `{{ cifmw_basedir }}/artifacts` instead